charts/incubator/invidious/values.yaml

```
@@ -3,13 +3,71 @@ image:
   pullPolicy: IfNotPresent
   tag: latest@sha256:3bb9e44e7d69baf91728c0a88115ef5fdb3adc91432b851e11e4d0e78bbe6e7a
 
-podSecurityContext:
-  runAsGroup: 0
-  runAsUser: 0
-
 securityContext:
-  readOnlyRootFilesystem: false
-  runAsNonRoot: false
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsGroup: 0
+    runAsUser: 0
+
+service:
+  main:
+    ports:
+      main:
+        port: 10594
+        protocol: http
+
+workload:
+  main:
+    podSpec:
+      initContainers:
+        dbseed:
+          type: install
+          enabled: true
+          imageSelector: image
```

```suggestion
          imageSelector: postgresClientImage
```
Probably will need this